test/help.test.ts

```diff
@@ -0,0 +1,35 @@
+
+import {expect, test} from '@oclif/test'
+
+describe('Help message handling', () => {
+  test
+    .stdout()
+    .command(['help'])
+  .it('works for global Mix', (ctx)=>{
```

Comment:
It's preferrable if a unit test description provides the complete context. That way, if a unit test fails, the unit test description of the failed test provides complete information.

Update the unit test description to "provides global help".